Documentation/building/unix/dependencies.md

```diff
@@ -28,6 +29,18 @@ to provide install instructions to obtain the missing dependency, e.g.:
 
     error : Could not find required program '7za'. Please run: brew install 'p7zip'.
 
+<a name="homebrew" />
+
+## Homebrew
+
+[Homebrew](https://brew.sh) must be installed and available via `$PATH` in
+order to provision xamarin-android.
+
+When building on M1 machines, use the **arch**(1) command to allow Homebrew
```

Comment:
I think this would be more future-proof once M2 and later processors come out:

```suggestion
When building on Apple Silicon (ARM64) machines, use the **arch**(1) command to allow Homebrew
```

Reply:
This change has been made.  Thanks!